src/components/Module.vue

```diff
@@ -1,72 +1,86 @@
       <div>
         <table>
-          <tr>
-            <th v-for="col in cols" v-bind:style="{ width: col.width + 'px' }">{{col.title}}</th>
-          </tr>
-          <tr v-for="param in module.params">
-            <td>{{param.Argument}}</td>
-            <td v-html="param.Type"></td>
-            <td>{{param.Description}}</td>
-          </tr>
+          <thead>
+            <tr>
+              <th v-for="col in cols" v-bind:style="{ width: col.width + 'px' }">{{col.title}}</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr v-for="param in module.params">
+              <td>{{param.Argument}}</td>
+              <td v-html="param.Type"></td>
+              <td v-html="param.Description"></td>
+            </tr>
+          </tbody>
         </table>
       </div>
       <div v-if="module.options !== null">
         <h4>Options</h4>
         <table>
-          <tr>
-            <th v-for="col in optionsCols" v-bind:style="{ width: col.width + 'px' }">{{col.title}}</th>
-          </tr>
-          <tr v-for="option in module.options">
-            <td>{{option.Prop}}</td>
-            <td>{{option.Type}}</td>
-            <td>{{option.Default}}</td>
-            <td>{{option.Description}}</td>
-          </tr>
+          <thead>
+            <tr>
+              <th v-for="col in optionsCols" v-bind:style="{ width: col.width + 'px' }">{{col.title}}</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr v-for="option in module.options">
+              <td>{{option.Prop}}</td>
+              <td v-html="option.Type"></td>
+              <td>{{option.Default}}</td>
+              <td>{{option.Description}}</td>
+            </tr>
+          </tbody>
         </table>
       </div>
       <div v-if="module.returns.length > 0">
         <h4>Returns</h4>
-        <p>{{module.returns[0].type}} - {{module.returns[0].desc}}</p>
+        <p v-html="module.returns[0].type + ' - ' + module.returns[0].desc"></p>
       </div>
       <div v-if="module.throws.length > 0">
         <h4>Throws</h4>
         <p>{{module.throws[0].type}} - {{module.throws[0].desc}}</p>
       </div>
-      <div v-if="module.snippet !== false">
-        <h4>Example</h4>
-          <prism language="javascript">{{module.snippet}}</prism>
-      </div>
     </Col>
     <Col span="7" offset="1">
-      <div v-if="module.hasMap">
-        <leaflet-map :code="module.example"></leaflet-map>
+      <div v-show="module.hasMap">
+        <div :id="'map_' + module.name"></div>
       </div>
       <p class="npmBadge">npm install {{module.npmName}}</p>
 
       <div v-if="module.parent !== null">
-        <p class="hasParent" >
+        <p class="hasParent">
           <strong>Note:</strong> {{module.name}} is part of the {{module.npmName}} module.<br><br>
           To use it as a stand-alone module will need to import {{module.npmName}} and call the {{module.name}} method.
         </p>
       </div>
     </Col>
+    <Col span="24">
```

Comment:
@rowanwins `@example` is now on it's own 24 span Column, noticed a lot of examples were getting "clipped" on smaller resolution browser windows.

This isn't a mobile-friendly solution, but it's a tad better on my tiny screen 😄